indico/modules/events/registration/templates/display/registration_modify.html

```diff
@@ -18,16 +18,19 @@
     {% if registration.can_be_modified or registration.invitation %}
         <div ng-app="nd" ng-controller="AppCtrl">
             <div nd-reg-form
-                conf-id="{{ event.id }}"
-                conf-sections="{{ sections | tojson | forceescape }}"
-                conf-currency="{{ registration.currency }}"
-                post-url="{{ request.url | tojson | forceescape }}"
-                check-email-url="{{ url_for('.check_email', regform) | tojson | forceescape }}"
-                registration-data="{{ registration_data | tojson | forceescape }}"
-                registration-meta-data="{{ registration_metadata | tojson | forceescape }}"
-                registration-uuid="{{ registration.uuid }}"
-                update-mode="true"></div>
+                 conf-id="{{ event.id }}"
+                 conf-sections="{{ sections | tojson | forceescape }}"
+                 conf-currency="{{ registration.currency }}"
+                 post-url="{{ request.url | tojson | forceescape }}"
+                 check-email-url="{{ url_for('.check_email', regform) | tojson | forceescape }}"
+                 registration-data="{{ registration_data | tojson | forceescape }}"
+                 registration-meta-data="{{ registration_metadata | tojson | forceescape }}"
+                 registration-uuid="{{ registration.uuid }}"
+                 update-mode="true">
+            </div>
         </div>
+        {{ template_hook('cern-access-personal-data', event=event, regform=registration.registration_form,
```

Comment:
we don't want to show this when editing a registration, just for the initial registration

Reply:
but feel free to trigger a signal anyway as long as you can check in the plugin code whether it's a new registration or editing an existing one